### PR TITLE
feat(): add missing types for indicator lights, fix(): wrong maxGear tag

### PR DESCRIPTION
**Pull request by tastydev**

### CODE REVIEW

**Review by LeonMrBonnie — CHANGES_REQUESTED** [review]
Please specify the indicatorLights property like all other properties

File: client/index.d.ts [hunk 1]
```diff
@@ -672,6 +672,13 @@ declare module "alt-client" {
      */
     public handling: IVehicleHandling;
 
+    /**
+     * Vehicle indicator lights
```
Comment (LeonMrBonnie) [inline client/index.d.ts:676]: Missing a `.` at the end
Maybe also add something like `state`

File: client/index.d.ts [hunk 2]
```diff
@@ -672,6 +672,15 @@ declare module "alt-client" {
      */
     public handling: IVehicleHandling;
 
+    /**
+     * Vehicle indicator lights.
+     *
+     * Set & get the state for indicator lights.
```
Comment (LeonMrBonnie) [inline client/index.d.ts:678]: Just remove this line, its not needed

**Review by LeonMrBonnie — APPROVED** [review]

**Review by Lhoerion — APPROVED** [review]
Thanks!

**Pull request merged**